### PR TITLE
docker-compose-swarm.yml has the wrong container name for kafka

### PR DESCRIPTION
**Pull request by dsevilla**
It read ${REPO}kafka-docker instead of ${REPO}kafka

### CODE REVIEW

File: docker-compose-swarm.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     ports:
       - "2181:2181"
   kafka:
-    image: ${REPO}kafka-docker:latest
+    image: ${REPO}kafka:latest
```
Comment (sscaling) [inline docker-compose-swarm.yml]: i'm not sure why this even uses the `${REPO}` environment var. It should just be the canonical `wurstmeister/kafka:latest` image

Comment (dsevilla) [reply docker-compose-swarm.yml]: I think it's there as a leftover of some tests using docker stacks. In stacks you usually build a temporal registry, and you have to tag the images something like hostname:port/kafka (e.g. 127.0.0.1:5000/kafka). Anyway, once the image is stable and available in the docker hub, no need for this REPO variable, so it should be removed in the github distribution.

File: docker-compose-swarm.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ services:
     ports:
       - "2181:2181"
   kafka:
-    image: ${REPO}kafka-docker:latest
+    image: kafka:latest
```
Comment (sscaling) [inline docker-compose-swarm.yml]: This should be `wurstmeister/kafka:latest` (so it references the docker hub build)

**Comment by sscaling** [comment]
Thanks!

**Pull request merged**